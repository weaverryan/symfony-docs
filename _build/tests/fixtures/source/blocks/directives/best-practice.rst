
.. best-practice::

    Use the bcrypt encoder for hashing your users' passwords.
