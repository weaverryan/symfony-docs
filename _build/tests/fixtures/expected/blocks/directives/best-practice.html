<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8" />
</head>
<body>
<div class="admonition-best-practice admonition-wrapper">
    <div class="best-practice"></div>
    <div class="admonition admonition-best-practice">
        <p class="admonition-title">Best Practice</p>
        <p>Use the bcrypt encoder for hashing your users' passwords.</p>
    </div>
</div>
</body>
</html>
